apps/judicial-system/web/src/components/FormProvider/FormProvider.tsx

```diff
@@ -91,29 +90,36 @@ const FormProvider = ({ children }: Props) => {
 
   const [state, setState] = useState<ProviderState>()
   const [caseId, setCaseId] = useState<string>()
+  const [pathname, setPathname] = useState<string>()
   const [workingCase, setWorkingCase] = useState<Case>({
     ...initialState,
     type: caseType,
     policeCaseNumbers: caseType === CaseType.INDICTMENT ? [''] : [],
   })
 
   // Used in exported indicators
-  const replacingCase = router.query.id && router.query.id !== caseId
-  const replacingPath = router.pathname !== window.location.pathname
+  const id = typeof router.query.id === 'string' ? router.query.id : undefined
```

Comment:
Could you please explain this change? I can't quite make sense of what this achieves.